torchani/units.py

```diff
@@ -0,0 +1,124 @@
+r"""Unit conversion factors
```

Comment:
Is this copied from ase, because you tried to import `ase.unit` and it failed?

Reply:
No, the thing is that ASE uses its own units, which are CODATA 2014, but they may change their units at some point. Before they used to use CODATA 1986 for some reason, and I don't think we want our units to change just because their units change, right?

Reply:
I would not want our units to depend on their code changing or not changing, this is why I copied.

Reply:
Also if we directly use ASE here then all of torchani will depend on ASE, which I also don't like very much. Right now only the code in ase.py and some tools/tests actually use ASE.

Reply:
Sure, that works